package.json

```diff
@@ -13,7 +13,7 @@
   "author": "Gant Laborde",
   "license": "MIT",
   "dependencies": {
-    "gluegun": "^0.22.0",
+    "gluegun": "^1.0.0",
```

Comment:
The new hawtness is on `infinitered/gluegun`.  `1.0.0` is current (aka `0.22.0`).

Reply:
Yup, let me know when I can point it at `next` and then I think I'll be good to roll with this PR :)

Reply:
Good to go dude!   `gluegun@next` .  Lemme know how it goes!